Resources/Prototypes/_Andromeda/voices.yml

```
@@ -0,0 +1,152 @@
+  nameId: highpitch+Ricardo
+  sex: Male
+  silicon: false
+  voice: 17
+
+- type: voice
+  id: higherpitch-ricardo
+  name: Ricardo mais fino
+  nameId: higherpitch+Ricardo
+  sex: Male
+  silicon: false
+  voice: 18
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)